Makefile

```diff
@@ -30,3 +32,17 @@ back/test-db: back
 .PHONY: back/run
 back/run:
 	stack exec --cwd back -- guide
+
+# Create a Docker image for the backend; will only work on Travis because
+# the binary has to have been compiled on Ubuntu Xenial (the OS used in the
+# Docker file)
+.PHONY: back/travis-docker
```

Comment:
If I run this command localy?

Reply:
It won't work unless you have Ubuntu Xenial. The reason for that is that it copies a locally built binary into the Docker image (which uses Ubuntu) and this binary has to be compatible with system libraries.

Reply:
A better way to do this would be building everything inside the container on Travis, but this can be fixed later.